lib/SlackBot.js

```diff
@@ -614,7 +625,12 @@ function Slackbot(configuration) {
 
                                             // Always update these because the token could become invalid
                                             // and scopes could change.
-                                            user.access_token = auth.access_token;
+                                            if (slack_botkit.config.encrypt) {
```

Comment:
More appropriate and readable format is: `slack_botkit.config.encryptionEnabled`